### PR TITLE
Fixup move gluon.metric api docs

### PR DESCRIPTION
**Pull request by leezu**
Fixup https://github.com/apache/incubator-mxnet/commit/cec86add654fccda2f2112d4487738854e7663ca

### CODE REVIEW

**Comment by leezu** [comment]
@mxnet-bot run ci [all]

File: docs/python_docs/python/api/gluon/metric/index.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+mxnet.metric
+============
+
+.. automodule:: mxnet.metric
```
Comment (szha) [inline docs/python_docs/python/api/gluon/metric/index.rst:21]: gluon.metric?

**Comment by leezu** [comment]
@mxnet-bot run ci [windows.gpu]

**Comment by leezu** [comment]
@mxnet-bot run ci [windows-gpu]

**Comment by leezu** [comment]
@mxnet-bot run ci [windows-gpu]

**Pull request merged**